### DISCUSSION TITLE
How to create org ID with time inclueded?

### DISCUSSION THREAD

**Discussion by VitalyAnkh** [nobiot/org-transclusion#100] [General]
Hi! The gif in README.md shows a org-mode file whose id includes its time when it's created. How to do that with org-roam v2? I use this:
```
(setq org-roam-capture-templates
  (quote (
    ;; Override default to set timestamp to UTC
    ("d" "default" plain (function org-roam--capture-get-point) "%?"
     :target "%(format-time-string \"%Y%m%dT%H%M%SZ-${slug}\" (current-time) t)"
     :head ":PROPERTIES:
:ID:    %(format-time-string \"%Y%m%dT%H%M%SZ\" (current-time) t)
:TITLE: ${title}
:END:
,#+ROAM_TAGS:
,#+ROAM_ALIAS:
"
     :unnarrowed t))))
```
But when I invoke `org-roam-node-insert`, it says 'Temples need to specify ":target" '.

**Comment by nobiot** [comment]
I think the easiest is to upgrade your Org mode to 9.5 and set this customizing variable to 'ts.

![Screenshot from 2021-11-02 08-17-30](https://user-images.githubusercontent.com/12507865/139802767-747e1102-0b3c-41cc-8a5d-32e17c34f0c8.png)

This question is not related to Org-transclusion and not really Org-roam. It's probably quicker for you to ask elsewhere (such as Reddit or StackExchange); Org-roam's Discourse community has discussed this before, so you should also be able to search for the exchange there..

If you need more detail on the method, you can also come to Org-roam's Discouse (or Slack).  I'm active on the Discourse forum, so I might see you there.

**Reply by VitalyAnkh** [reply]
Thanks! Sorry for my irrelevant question.

**Reply by nobiot** [reply]
There is no problem asking. It's just that you might not get answers as quickly as you might expect. Like I said, it would be quicker for you if you know the right community.
